packages/gatsby/src/utils/browserslist.ts

```diff
@@ -15,7 +16,7 @@ function installedGatsbyVersion(directory) {
   }
 }
```

Comment:
a little nit. This function was moved to gatsby-core-utils. Could you delete this and import from there. The only difference is L13 is unique to this file.